### ISSUE TITLE
Understanding Overrides

### ISSUE THREAD

**Issue by zachdaniel** [team-alembic/ash_authentication_phoenix#80]
I think we need to come up with some way to explain what overrides are available in general. The simple answer may be to direct people to the source code of the default overrides so they can see how we are doing what we're doing.

**Comment by jimsynz** [comment]
Each component documents it's overrides in the moduledocs, but I get that people may not be looking there, eg: https://ash-hq.org/docs/module/ash_authentication_phoenix/1.4.5/ashauthentication-phoenix-components-oauth2

**Comment by zachdaniel** [comment]
Yeah, the difficulty is not knowing where to look. A guide that has a link to each override and does a bit of exposition on "here is how you do custom UI stuff, and if you need structural changes then you can make your own page and use our components..." that kind of thing. I think guides are the most discoverable thing we have at the moment.

**Comment by jimsynz** [comment]
Agreed.

**Comment by frankdugan3** [comment]
After stumbling through this myself, I found the easiest thing to do was to look at the [default overrides](https://github.com/team-alembic/ash_authentication_phoenix/blob/main/lib/ash_authentication_phoenix/overrides/default.ex). A link to that in the docs would help a lot.

**Comment by dmitriid** [comment]
It's also unclear where and how you would use these overrides.

Since the guide only goes through autogenerated routes, the next question is... "Now what?" :)

**Comment by dmitriid** [comment]
It also doesn't help that the names don't fit on screen in docs :) Though I don't know how to fix that

![image](https://github.com/team-alembic/ash_authentication_phoenix/assets/32768/cc993b53-f1f0-4a03-af08-78167aa27ccc)

**Comment by jimsynz** [comment]
Closing - overrides are now documented in the [UI overrides guide](https://hexdocs.pm/ash_authentication_phoenix/ui-overrides.html).
